### DISCUSSION TITLE
Error CUDA_ARCHITECTURE is empty

### DISCUSSION THREAD

**Discussion by AdriAmaro** [NVlabs/instant-ngp#259] [Q&A]
Hello, I have an error when creating the project with cmake. is next

`C:\Users\Adrian\source\repos\instant-ngp>cmake . -B build
-- The C compiler identification is GNU 10.2.0
-- The CXX compiler identification is GNU 10.2.0
-- The CUDA compiler identification is unknown
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting CUDA compiler ABI info
CMake Error in /cygdrive/c/Users/Adrian/source/repos/instant-ngp/build/CMakeFiles/CMakeTmp/CMakeLists.txt:
  CUDA_ARCHITECTURES is empty for target "cmTC_04d61".

CMake Error in /cygdrive/c/Users/Adrian/source/repos/instant-ngp/build/CMakeFiles/CMakeTmp/CMakeLists.txt:
  CUDA_ARCHITECTURES is empty for target "cmTC_04d61".

CMake Error at /usr/share/cmake-3.20.0/Modules/CMakeDetermineCompilerABI.cmake:49 (try_compile):
  Failed to generate test project build system.
Call Stack (most recent call first):
  /usr/share/cmake-3.20.0/Modules/CMakeTestCUDACompiler.cmake:19 (CMAKE_DETERMINE_COMPILER_ABI)
  CMakeLists.txt:11 (PROJECT)

-- Configuring incomplete, errors occurred!
See also "/cygdrive/c/Users/Adrian/source/repos/instant-ngp/build/CMakeFiles/CMakeOutput.log".
See also "/cygdrive/c/Users/Adrian/source/repos/instant-ngp/build/CMakeFiles/CMakeError.log".`

I have created the environment variable CUDA_ARCHITECTURE = 61

but it keeps showing up.

I use Windows 11, i7 of 8, 8 ram, GTX 1050

**Comment by suwcg** [comment]
I come across the same question. Do you solved?
